virttest/shared/unattended/RHEL-8-devel.ks

```diff
@@ -107,5 +108,20 @@ EOF
 cat >> '/home/test/.bashrc' << EOF
 alias shutdown='sudo shutdown'
 EOF
+# Install and lock packages specified via 'kickstart_instlock_pkgs' parameter
+install_and_lock()
+{
+    for PKG in KVM_TEST_PKGS; do
+        dnf in $PKG -y --nogpgcheck
```

Comment:
Prefer to use the full command 'install' to replace the alias `in`.

Reply:
Better `ECHO` the specified pkg installation,  it is helpful for debugging i think.

Reply:
Thanks for pointing that out. Actually I think i use `in` all the time without any problems, but I agree let's do it safer and change it into `install`.